### ISSUE TITLE
Word type meanings?

### ISSUE THREAD

**Issue by colelamers** [AyeshJayasekara/English-Dictionary-SQLite#1]
I'm trying to decipher what some of the word types are but some of them don't make sense. For example, when I look at the word "blindfold", it returns "a." and "v. t." for the 2 results of that word. V. T. sounds to me to be a Transitive Verb, but A doesn't make sense. The definitions do not help either because they both seems like verbs.

I've been trying to use this: https://en.wikipedia.org/wiki/List_of_glossing_abbreviations as guidance but it has not been very helpful. So providing the key to this database would be most helpful.

**Comment by Ritika-Das** [comment]
I think "a." refers to adjectives.

**Comment by alaymari** [comment]
I do not think so. The db has entries with `adj.` tag which the regular dictionaries use to mean adjective. 

It would be nice if we could get the description of all the tags used.

**Comment by bw-varun** [comment]
https://en.wiktionary.org/wiki/Appendix:List_of_glossing_abbreviations
